src/citrine/informatics/predictor_evaluation_metrics.py

```diff
@@ -82,6 +84,24 @@ def __str__(self):
         return "NDME"
 
 
+class RSquared(Serializable["RSquared"], PredictorEvaluationMetric):
+    """Fraction of variance explained, commonly known as R^2.
+
+    This dimensionless metric is equal to 1 - (mean squared error / variance of data).
+    It is equal to the coefficient of determination calculated with respect to the line
+    `predicted = actual`, hence it is commonly referred to as R^2. But unlike R^2 from
+    ordinary linear regression, this metric can be negative.
+    """
+
+    typ = properties.String("type", default="RSquared", deserializable=False)
+
+    def __repr__(self):
+        return "r_squared"
```

Comment:
In the backend, I'm pretty sure this is "R^2":
https://github.com/CitrineInformatics/mithril/blob/master/src/main/scala/io/citrine/nucleus/ml/analysis/crossvalidation/evaluators/metrics/RegressionMetrics.scala#L43

Reply:
Good point--I didn't actually realize that those keys in mithril became the keys of the "metrics" dictionary.